bin/archive-old-enquiries

```diff
@@ -43,6 +43,7 @@ my ($opts, $usage) = describe_options(
     ['closure_text=s', "text of the message to add to reports" ],
     ['closed_state=s', "state to set reports to" ],
     ['limit|l=s',      "limit to a certain number of reports/users to be closed"],
+    ['show_emails|s', "output emails that would be sent to STDOUT and prevents closure. Overrides --commit / -c."],
```

Comment:
Given this help message, I think update_options should presumably set commit to 0 if show_emails is set, to guarantee this. Then you wouldn't need to add a line to close_problems.